resources/views/customer/customer-show.twig

```diff
@@ -0,0 +1,98 @@
+{% extends "base.twig" %}
+
+{% block body %}
+<h1>{{ customer.company_name }}</h1>
+<form class="form-horizontal">
+    <div class="row">
+        <label class="control-label col-xs-8 col-sm-2">
+            VAT number
+        </label>
```

Comment:
Un `label` sans un `input` associé est un bête `span`.

https://www.w3.org/TR/2016/REC-html51-20161101/sec-forms.html#the-label-element